npm/qsharp/src/compiler/compiler.ts

```diff
@@ -77,6 +78,8 @@ export interface ICompiler {
     exerciseSources: string[],
     eventHandler: IQscEventTarget,
   ): Promise<boolean>;
+
+  collectTestCallables(program: IProgramConfig): Promise<string[]>;
```

Comment:
Totally a nitpick but I feel like `getTestCallables` is a simpler and nicer name.

Reply:
I chose `collect` because it is actually performing iteration and collection logic, while `get` implies getter-style functionality (just reading a field, little to no computation). I'm happy to switch to `get` if there's a compelling reason though

Reply:
Only because `getHir`, `getAst` , `getEstimates` etc exist (and they're all doing more than "get"ting too). My only ask is that you take a quick look at the overall public API in this file and decide for yourself what name would fit best.

Reply:
Alright, that convinces me, I like consistency.